Sources/FluentBenchmark/FluentBenchmarker.swift

```diff
@@ -275,7 +276,7 @@ public final class FluentBenchmarker {
         ]) {
             struct PlanetJSON: Codable, Equatable {
                 struct GalaxyID: Codable, Equatable {
-                    var id: Int
+                    var galaxy_id: Int
```

Comment:
What's the reasoning behind this change?

Reply:
This has to do with the parent key for `Planet.galaxy` is `galaxy_id`. When you encode a `Planet` as JSON, the JSON key for the galaxy ID is `galaxy_id`.

```swift
func encode(to encoder: Encoder) throws {
    var container = encoder.singleValueContainer()

    if case let .loaded(parent) = self.eagerLoadedValue {
        try container.encode(parent)
    } else {
        try container.encode([self.$id.key: self.id])
    }
}
```

Reply:
By the way, this is the way things are currently done with separate `Parent` types.